### PR TITLE
Update gulp-typescript.d.ts to use tsconfig.json

### PR DESCRIPTION
**Pull request by SergioMorchon**
As seen at https://github.com/ivogabe/gulp-typescript/blob/master/lib/project.ts

### CODE REVIEW

**Comment by vvakame** [comment]
@pspeter3 could you review this PR?

File: gulp-typescript/gulp-typescript.d.ts [hunk 1]
```diff
@@ -9,7 +9,30 @@ declare module "gulp-typescript" {
     function GulpTypescript(params: GulpTypescript.Params, filters?: GulpTypescript.FilterSettings, reporter?: GulpTypescript.Reporter): GulpTypescript.CompilationStream;
 
     module GulpTypescript {
-        export function createProject(params: Params): Params;
+        export interface TsConfig {
```
Comment (pspeter3) [inline gulp-typescript/gulp-typescript.d.ts]: It seems reasonable to depend on the the TypeScript type definition here.

**Comment by pspeter3** [comment]
lgtm when it passes tests.

**Comment by mhegazy** [comment]
Can you please fix the failing Travis CI tests and update the PR!

**Comment by RyanCavanaugh** [comment]
Hello and thank you for your contribution!

Due to an excessively long queue of pull requests that have become stale, we are "declaring bankruptcy" and closing all PRs opened before May 1, 2016. If you'd still like to merge this code in, please open a new PR that has been merged and rebased with the `master` branch.

Going forward, we are committing to review or merge all PRs on a regular basis so this bankruptcy will not occur again. We apologize for the incovenience and hope you will continue to contribute to DefinitelyTyped in the future.

Thanks
The TypeScript and DefinitelyTyped teams

**Pull request closed without merging**